src/main/resources/templates/pollingunits.html

```diff
@@ -13,7 +21,7 @@
 	<body>
     <h2>Polling Units</h2>
     <div>
-    <table>
+    <table class="striped responsive-table">
```

Comment:
@dinobi @benfluleck This html page is the static version that displays a table of records that is pageable and sortable.  Deployed at https://naijagov.herokuapp.com/pollingunits